scripts/asmCheck_tracer.py

```diff
@@ -1,25 +1,28 @@
-import sys
 import argparse
 
-parser = argparse.ArgumentParser(description="Match SST output with asm file and/or print call stack at a specified clock tick - use with TRACER_ON in target program.  Use 'riscv64-unknown-elf-objdump -dC -Mno-aliases <exe>' to create assembly file")
+parser = argparse.ArgumentParser(description="""\
+Match SST output with asm file and/or print call stack at a specified clock
+tick - use with TRACER_ON in target program.
+
+Use 'riscv64-unknown-elf-objdump -dC -Mno-aliases <exe>' to create assembly file""")
```

Comment:
`"""` starts and ends a multi-line string, which can use `\` to suppress newlines but otherwise includes them. This avoids creating an ultra-long source line.